Sources/Sentry/SentryFrameInAppLogic.m

```diff
@@ -0,0 +1,44 @@
+#import "SentryFrameInAppLogic.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface
+SentryFrameInAppLogic ()
+
+@property (nonatomic, copy, readonly) NSArray<NSString *> *inAppIncludes;
+@property (nonatomic, copy, readonly) NSArray<NSString *> *inAppExcludes;
+
+@end
+
+@implementation SentryFrameInAppLogic
+
+- (instancetype)initWithInAppIncludes:(NSArray<NSString *> *)inAppIncludes
+                        inAppExcludes:(NSArray<NSString *> *)inAppExcludes
+{
+    if (self = [super init]) {
+        _inAppIncludes = inAppIncludes;
+        _inAppExcludes = inAppExcludes;
+    }
+
+    return self;
+}
+
+- (BOOL)isInApp:(NSString *)imageName
```

Comment:
L: is `imageName` a nullable field? on Java, if `imageName` is null or empty, we mark it as `inApp`, but my comment is more about if it could throw a null pointer or not

Reply:
same thing for `lastPathComponent`

Reply:
Excellent point. I think it should be `nullable`.